src/utilities/boleto/index.ts

```diff
@@ -134,3 +140,40 @@ export function format(boleto: string) {
       return result;
     }, '');
 }
+
+export function getValueInCents(digitableLine: string): number {
+  if (!digitableLine || !isValid(digitableLine)) return 0;
+
+  const digits = onlyNumbers(digitableLine);
+
+  const valueStartIndex = digits.length - 10;
```

Comment:
what do you think to create a variable to store the "10" value and give it a name?